spec-grpc/src/main/java/io/a2a/grpc/utils/ProtoUtils.java

```diff
@@ -143,8 +145,12 @@ public static io.a2a.grpc.Task task(Task task) {
         public static io.a2a.grpc.Message message(Message message) {
             io.a2a.grpc.Message.Builder builder = io.a2a.grpc.Message.newBuilder();
             builder.setMessageId(message.getMessageId());
-            builder.setContextId(message.getContextId());
-            builder.setTaskId(message.getTaskId());
+            if (message.getContextId() != null) {
+                builder.setContextId(message.getContextId());
+            }
+            if (message.getTaskId() != null) {
+                builder.setTaskId(message.getTaskId());
+            }
```

Comment:
I think the corresponding `message` method in `FromProto` will need to be updated so the value is only set if non empty.